### PR TITLE
BFD-2154: Removes vault prefix on locust tests

### PR DESCRIPTION
**Pull request by cbrunefearless**
**JIRA Ticket:**
[BFD-2154](https://jira.cms.gov/browse/BFD-2154)

**User Story or Bug Summary:**

The locust test suites were developed ahead of a complete initial implementation of BFD's usage of AWS SSM Parameter Store. As a result, it's relying on keys that should no longer be maintained. This task is to update the references to vault-prefixed hierarchies, i.e. use `data_server_db_username` in place of `vault_data_server_db_username` and `data_server_db_password` in place of `vault_data_server_db_password` wherever used.

AC:

locust no longer relies on vault-prefixed server database credentials
vault-prefixed database credentials are removed from the base sensitive configurations in all three environments test, prod-sbx, and prod

---

### What Does This PR Do?

Removes vault  prefix on locust tests.

### What Should Reviewers Watch For?

If you're reviewing this PR, please check for these things in particular:

* Verify all PR security questions and checklists have been completed and addressed.

### What Security Implications Does This PR Have?

Submitters should complete the following questionnaire:

* If the answer to any of the questions below is **Yes**, then **you must** supply a link to the associated Security Impact Assessment (SIA), security checklist, or other similar document in Confluence here: **N/A**

    * Does this PR add any new software dependencies? 
      * [ ] Yes
      * [X] No
    * Does this PR modify or invalidate any of our security controls?
      * [ ] Yes
      * [X] No
    * Does this PR store or transmit data that was not stored or transmitted before?
      * [ ] Yes
      * [X] No

* If the answer to any of the questions below is **Yes**, then please add @StewGoin as a reviewer, and note that this PR **should not be merged** unless/until he also approves it.
    * Do you think this PR requires additional review of its security implications for other reasons?
      * [ ] Yes
      * [X] No

### What Needs to Be Merged and Deployed Before this PR?

This PR cannot be either merged or deployed until the following prerequisite changes have been fully deployed:

* N/A

### Submitter Checklist

I have gone through and verified that...:

* [X] I have named this PR and branch so they are [automatically linked](https://confluence.atlassian.com/adminjiracloud/integrating-with-development-tools-776636216.html) to the (most) relevant Jira issue. Ie: `BFD-123: Adds foo`
* [X] This PR is reasonably limited in scope, to help ensure that:
    1. It doesn't unnecessarily tie a bunch of disparate features, fixes, refactorings, etc. together.
    2. There isn't too much of a burden on reviewers.
    3. Any problems it causes have a small "blast radius".
    4. It'll be easier to rollback if that becomes necessary.
* [X] This PR includes any required documentation changes, including `README` updates and changelog / release notes entries.
* [X] The data dictionary has been updated with any field mapping changes, if any were made.
* [X] All new and modified code is appropriately commented, such that the what and why of its design would be reasonably clear to engineers, preferably ones unfamiliar with the project.
* [X] All tech debt and/or shortcomings introduced by this PR are detailed in `TODO` and/or `FIXME` comments, which include a JIRA ticket ID for any items that require urgent attention.
* [X] Reviews are requested from both:
    * At least two other engineers on this project, at least one of whom is a senior engineer or owns the relevant component(s) here.
    * Any relevant engineers on other projects (e.g. DC GEO, BB2, etc.).
* [X] Any deviations from the other policies in the [DASG Engineering Standards](https://github.com/CMSgov/cms-oeda-dasg/blob/master/policies/engineering_standards.md) are specifically called out in this PR, above.
    * Please review the standards every few months to ensure you're familiar with them.

### CODE REVIEW

File: runbooks/README.md [hunk 1]
```diff
@@ -2,17 +2,26 @@
 Table of Contents
 =================
 
-* [How to Re-run a Failed BFD Pipeline Load](./how-to-rerun-failed-pipeline-load.md#how-to-re-run-a-failed-bfd-pipeline-load)
```
Comment (lsmitchell) [inline runbooks/README.md:5]: There was another PR where the order of this file changed a bunch, whats the deal with that?

**Review by lsmitchell — APPROVED** [review]

**Review by mjburling — CHANGES_REQUESTED** [review]
Could you make sure those vault-prefixed values are removed from the sensitive eyaml files as part of this change set?

File: runbooks/README.md (hunk 1 above)
Comment (cbrunefearless) [reply runbooks/README.md:5]: This was from github actions.. Very strange honestly.. I committed my files and than github actions was like, oh here u go :)

**Comment by cbrunefearless** [comment]
> Could you make sure those vault-prefixed values are removed from the sensitive eyaml files as part of this change set?

yes sir!

**Review by mjburling — DISMISSED** [review]
This works. Thanks, @cbrunefearless!

**Review by brianburton — APPROVED** [review]

**Review by malessi — APPROVED** [review]

**Review by mjburling — APPROVED** [review]

**Review by malessi — APPROVED** [review]

**Pull request merged**